torch/sparse/__init__.py

```diff
@@ -1,53 +1,118 @@
 import torch
 from torch import _C
-
-import sys
+from .._utils import _type, _cuda
 
 _sparse_tensor_classes = set()
 
 
-class DoubleTensor(_C.SparseDoubleTensorBase):
+class _SparseTensorBase(object):
```

Comment:
Can you use _TensorBase as the base class instead and use a mix-in? (Like in `cuda/__init__.py`  _CudaBase)